wp-content/themes/wp-starter/inc/icons.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Custom Button Icons.
+ *
+ * @package WPStarter
+ */
+
+/**
+ * Add custom icons to the button icons array.
+ * */
+
+add_filter(
+	'acfbt_button_icons',
```

Comment:
Double check this hook, it might need to be updated to `acfbt_block_icons`, but I can't remember if I changed this hook.

Also, maybe we should put some instructions in the comments (lines 8-10) explaining how to add custom icons and where they are located.

Reply:
`acfbt_button_icons` is the hook we are using in the plugin. Also added some instructions!